gremlin-python/src/main/python/tests/driver/test_client.py

```diff
@@ -27,10 +27,13 @@
 from gremlin_python.process.strategies import OptionsStrategy
 from gremlin_python.structure.graph import Graph
 from gremlin_python.driver.aiohttp.transport import AiohttpTransport
+from gremlin_python.statics import *
```

Comment:
best practice is to import specific items
```suggestion
from gremlin_python.statics import bigint, long
```

Reply:
I see the other test just imports * already. Perhaps we can ignore this comment since the other tests don't consider that.